wp-admin/js/widgets/media-widgets.js

```diff
@@ -1,5 +1,5 @@
 /* eslint consistent-this: [ "error", "control" ] */
-wp.mediaWidgets = ( function( $ ) {
+wp.mediaWidgets = ( function( $, _wpMediaViewsL10n ) {
```

Comment:
Instead of accessing this “private” global variable, should we not instead access `wp.media.view.l10n`?